.github/workflows/coverage.yml

```diff
@@ -14,33 +14,21 @@ jobs:
         os: [ ubuntu-latest ]
 
     steps:
-      - name: Git Checkout
-        uses: actions/checkout@v3
+      - name: Git checkout
+        uses: actions/checkout@v4
         with:
           submodules: recursive
-      - name: Rust Toolchain
-        uses: actions-rs/toolchain@v1
+      - name: Install Rust toolchain
+        uses: dtolnay/rust-toolchain@stable
         with:
-          profile: minimal
-          toolchain: stable
           # We need this component to generate source coverage in stable
           components: llvm-tools-preview
-          override: true
-      # Wrapper for running coverage with -C instrument-coverage
-      - name: Cargo Install Coverage Wrapper
-        uses: actions-rs/cargo@v1
-        with:
-          command: install
-          args: cargo-llvm-cov
-      - name: Cargo Test w/ Coverage
-        uses: actions-rs/cargo@v1
-        with:
-          command: llvm-cov
-          args: --verbose --workspace --all-features --no-fail-fast --codecov --output-path codecov.json
-      - name: Codecov Upload
+      - name: Install `cargo-llvm-cov`
+        run: cargo install cargo-llvm-cov
+      - name: Cargo test w/code coverage
+        run: cargo llvm-cov --verbose --workspace --all-features --no-fail-fast --codecov --output-path codecov.json
+      - name: Codecov upload
         uses: codecov/codecov-action@v4
-        env:
-          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
🗺️ Removing this as #852 added `with: token` on line 35 below (hidden by default in this diff).

https://github.com/amazon-ion/ion-rust/pull/853/files#diff-a2115d277b5ca5a2f09a999e53440839cf332b94da177f3d1766334555b0f7c6L44-L47